Sources/Jetworking/Logger/Default/DefaultLogger.swift

```diff
@@ -0,0 +1,12 @@
+import Foundation
+import os.log
+
+final class DefaultLogger: Logger {
+    func log(_ message: String) {
```

Comment:
Nice! You could pass DefaultLogger instance in the interceptors initializer.

Reply:
Done.